components/tray/metadata/mods.md

```diff
@@ -4,6 +4,7 @@
 | `--mod-tray-bottom-to-content-area`         |
 | `--mod-tray-corner-radius`                  |
 | `--mod-tray-corner-radius-portrait`         |
+| `--mod-tray-edge-to-content-area`           |
```

Comment:
Same question like popover- does this need to go in the `tray.yml`? Are we supporting that file still- isn't just for the docs site?